### ISSUE TITLE
Download position in queue

### ISSUE THREAD

**Issue by kevin-stuart** [ziahamza/webui-aria2#178]
Is it possible to associate a position for a new download and change the position in the queue of an existing download? The aria2.addUri method has a position argument and there is a aria2.changePosition method as well.

**Comment by Jonathan34** [comment]
yes that's a good idea. i wanted to change the order in the queue but could not :(
